.github/workflows/verify-release.yml

```diff
@@ -0,0 +1,33 @@
+name: Reproducible Builds
+on:
+  # On tags
+  push:
+    tags:
+      - '*'
+  # Or manually
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to verify (>= 4.0.0-beta.1)'
```

Comment:
That's not normally how they are numbered. The first beta should have been 4.0.0-beta.0.

Reply:
What's the source for this naming strategy?

Reply:
Official sources allow for more wiggle room in pre release versions. However, it's moot because this is how release-it works by default and it's not really changeable.

Reply:
As discussed in the meeting, will update so the next release is `.2`. This may require manual edits to the package.json before initial pre-releases, but we can do that.